src/config/session-management.md

```diff
@@ -52,3 +52,12 @@ non-root users to be able to access the seatd session, add them to the `_seatd`
 group.
 
 Note that, unlike elogind, seatd doesn't do anything besides managing seats.
+
+## XDG_RUNTIME_DIR
+
+Install [elogind](#elogind) as your session manager to automatically setup
+`XDG_RUNTIME_DIR`.
+
+Alternatively, manually set the environment variable through the shell. Make
+sure to create a dedicated user directory and set its permissions to `700`. A
+good default location is `/run/user/$(id -u)`.
```

Comment:
The new section is missing an introduction explaining that it is an environment variable. Maybe link an official page about XDG spec, if such a thing exists.